packages/terra-menu/src/Menu.scss

```diff
@@ -0,0 +1,104 @@
+    border-top-color: var(--terra-menu-submenu-background-color, #f9f9f9);
+  }
+
+  .divider {
+    border-color: var(--terra-menu-divider-color, #dedfe0);
+    border-style: solid;
+    border-width: 1px 0 0;
+    margin-top: 2px;
+
+    &::after {
+      border-color: var(--terra-menu-divider-shadow-color, #f9f9f9);
+      border-style: solid;
+      border-width: 3px 0 0;
+      content: '';
+      display: block;
+      margin-bottom: 4px;
+    }
+  }
+
+  .submenu .divider::after {
+    border-color: var(--terra-menu-submenu-divider-shadow-color, #fff);
+  }
+
+  .header {
+    background-color: var(--terra-menu-submenu-header-background-color, #fff);
+    border-bottom: var(--terra-menu-submenu-header-border-bottom, 1px solid #dedfe0);
+    width: 100%;
+
+    .header-title {
+      color: var(--terra-menu-submenu-header-title-color, #1c1f21);
+      font-size: var(--terra-menu-submenu-header-title-font-size, 12px);
+      font-weight: normal;
+      line-height: var(--terra-menu-submenu-header-title-line-height, 1.1667);
+      margin: var(--terra-menu-submenu-header-title-margin, 7px 0 7px 0);
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+      word-break: normal;
+      word-wrap: normal;
+    }
+
+    .header-button {
+      background: 0 0;
+      border: 0;
+      color: var(--terra-menu-submenu-header-buttons-icon-color, #1c1f21);
+      cursor: pointer;
+      font-size: var(--terra-menu-submenu-header-buttons-icon-font-size, 12px);
+      font-weight: bold;
+      margin: var(--terra-menu-submenu-header-buttons-margin, 0);
+      padding: var(--terra-menu-submenu-header-buttons-padding, 8px);
+
+      &:hover {
+        color: var(--terra-menu-submenu-header-buttons-hover-color, #111);
+      }
+    }
+  }
+
+  .fullscreen {
+    .header {
+      background-color: var(--terra-menu-fullscreen-header-background-color, #fff);
+      border-bottom: var(--terra-menu-fullscreen-header-border-bottom, 1px solid #dedfe0);
+    }
+
+    .header-title {
+      color: var(--terra-menu-fullscreen-header-title-color, #1c1f21);
+      font-size: var(--terra-menu-fullscreen-header-title-font-size, 1rem);
+      line-height: var(--terra-menu-fullscreen-header-title-line-height, 1.2);
+      margin: var(--terra-menu-fullscreen-header-title-margin, 0.5714rem 0 0.5714rem 0.3571rem);
+    }
+
+    .header-button {
+      font-size: var(--terra-menu-fullscreen-header-buttons-icon-font-size, 1.45rem);
+      margin: var(--terra-menu-fullscreen-header-buttons-margin, 0.3571rem 0);
+      padding: var(--terra-menu-fullscreen-header-buttons-padding, 4px);
+    }
+
+    .header-button:first-child {
+      margin: 0;
+    }
+  }
+
+  .main-menu:not(.fullscreen) .header {
+    display: none;
+  }
+
+  .submenu:not(.fullscreen) .close-button {
+    display: none;
+  }
+
+  .list {
+    padding: 6px 0;
+  }
+
+  .main-menu .content {
+    background-color: var(--terra-menu-background-color, #fff);   
```

Comment:
Typically, it's best practice to set the color if you set the background color. We should allow theming for the foreground of the element as well.